dataverse-persistence/src/test/java/edu/harvard/iq/dataverse/persistence/ScriptRunner.java

```diff
@@ -0,0 +1,100 @@
+package edu.harvard.iq.dataverse.persistence;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.io.Reader;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+/**
+ * Runner of sql scripts
+ * <p>
+ * Based on MyBatis2 ScriptRunner
+ * 
+ * @see https://github.com/mybatis/ibatis-2/blob/master/src/main/java/com/ibatis/common/jdbc/ScriptRunner.java
+ */
+@Stateless
+public class ScriptRunner {
```

Comment:
Maybe SqlScriptRunner? if we anticipate running bash/perl scripts in the future too, then I would leave it.

Reply:
ok. I will change it